src/postman/operation.ts

```diff
@@ -81,12 +80,10 @@ function transformItem(item: Item, securitySchemes: PostmanSecurityScheme[]): IH
     path: getPath(item.request.url),
     summary: item.name,
     request,
-    responses: item.responses
-      .all()
-      .reduce<IHttpOperationResponse[]>(
-        (merged, response) => mergeResponses(merged, [transformResponse(response)]),
-        [],
-      ) as IHttpOperation['responses'],
+    responses: item.responses.all().reduce<IHttpOperation['responses']>(
+      (merged, response) => mergeResponses(merged, [transformResponse(response)]),
+      [] as any, // @todo do we want to load operations without responses?
```

Comment:
@karol-maciaszek Yes, I think we should load them anyway so that tooling (Prism or Platform) can show meaningful errors, such as "no responses found, but these are the one in the document:…"

Reply:
Sounds good, I'll remove the comment and let's proceed!